cpt-dl/src/cptdl/catalog.py

```diff
@@ -197,6 +197,10 @@
 	'CHILESTATIONS.PRCP': "http://iridl.ldeo.columbia.edu/home/.xchourio/.ACToday/.CHL/.prcp/T/{int(lead_high-lead_low)+1}/seasonalAverage/T/%28{targetcenter(target)}%29/VALUES/{int(lead_high-lead_low)+1}/mul/-999/setmissing_value/{'%5B%5D%5BT%5D' + filetype if filetype == 'cptv10.tsv' else 'data.nc'}",
 	'CHILESTATIONS.TMAX': "http://iridl.ldeo.columbia.edu/home/.xchourio/.ACToday/.CHL/.tempX/T/{int(lead_high-lead_low)+1}/seasonalAverage/T/%28{targetcenter(target)}%29/VALUES/-999/setmissing_value/{'%5B%5D%5BT%5D' + filetype if filetype == 'cptv10.tsv' else 'data.nc'}",
 	'CHILESTATIONS.TMIN': "http://iridl.ldeo.columbia.edu/home/.xchourio/.ACToday/.CHL/.tempN/T/{int(lead_high-lead_low)+1}/seasonalAverage/T/%28{targetcenter(target)}%29/VALUES/-999/setmissing_value/{'%5B%5D%5BT%5D' + filetype if filetype == 'cptv10.tsv' else 'data.nc'}",
+	'CHILE2STATIONS.PRCP': "https://iridl.ldeo.columbia.edu/SOURCES/.Chile/.DMC/.monthly_prec/dup/a%3A/.lat//name//Y/def/%3Aa%3A/.alt//units//ids/def//name//elev/def/%3Aa%3A/.lon//name//X/def/%3Aa%3A/.prcp/T/%28{target}%20{first_year}-{final_year}%29/seasonalAverage/-999/setmissing_value//name//prcp/def/%3Aa/%7BY/elev/X/prcp%7Dds/{'%5BSTNID%5D%5BT%5D/' + filetype if filetype == 'cptv10.tsv' else 'data.nc'}",
```

Comment:
Why is there stuff like `lat /name /Y def` in the URL? Shouldn't variable names get standardized in the catalog entry?

Reply:
Rémi explained that on https://bitbucket.org/iridl/dlentries/pull-requests/599/diff#Lentries/Chile/DMC/monthly_prec/index.texT71 .